src/components/modals/WalletListMenuModal.js

```diff
@@ -57,6 +66,13 @@ const getWalletOptions = async (params: { walletId: string, walletName?: string,
 
   const result = []
 
+  const splittable = await account.listSplittableWalletTypes(walletId)
+
+  if (splittable.length) {
```

Comment:
This is not a boolean, so you should use exact comparison, something like: 
`splittable.length !== null && splittable.length > 0`